exercises/leap/src/example.c

```diff
@@ -5,10 +5,10 @@ bool is_leap_year(int year)
    if (year % 4 == 0) {
 
       if (year % 400 == 0) {
-         return true;
+	 return true;
```

Comment:
I don't know why it looks wrong, but that line begins with a tab and a space which should be the correct indentation.